### DISCUSSION TITLE
UI is blurry on Linux when using Wayland

### DISCUSSION THREAD

**Discussion by viggy96** [f1multiviewer/issue-tracker#417] [Feature requests] [answered]
This is caused by using an older version of Electron.
Current workaround is to launch using the following arguments:
`f1multiviewer --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations`

**Comment by erikdrenth** [comment]
Sending in +1 to this one. I'd like to use Wayland on Fedora as I have to use fractional scaling on my laptop. Wayland handles that a lot better than X.  Launching with above settings in my .desktop file for muvi works fine. However, this breaks the ability to position windows, so I reverted to x(-wayland). Hope proper wayland-support can be fixed in the future.

**Comment by LapsTimeOFF** [comment, marked as answer]
Hey, is this still relevant? If so please open an issue in the issues tab or contact us on [Discord](https://muvi.gg/discord)
